aggregator/test/helpers/Fixture.ts

```diff
@@ -89,6 +89,7 @@ export default class Fixture {
     public chainId: number,
     public walletService: WalletService,
   ) {
+    // TODO How do we want to read in these addresses from a file? What defualt should we use?
```

Comment:
> What defualt should we use?

My preference is for no default. Startup should fail to start if the config is missing a required address.